src/mumble/DBus.cpp

```diff
@@ -84,6 +84,15 @@ void MumbleDBus::focus() {
 	g.mw->activateWindow();
 }
 
+void MumbleDBus::setTransmitMode(unsigned int mode) {
+	g.mw->qcbTransmitMode->setCurrentIndex(mode);
+	emit g.mw->qcbTransmitMode->activated(mode);
+}
```

Comment:
There should be a check whether the given mode is valid and if not, this function should print an error (probably using `qCritical`) and return

Reply:
I looked at the other DBus methods, which send an error back via DBus and copied that.